commands/betflip.js

```diff
@@ -11,36 +15,42 @@ module.exports = {
     owner_only: false,
     affect_config: true,
     action: function(message, config) {
-        user = update_dabs(message.author, config)
         content = message.content.toLowerCase().split(" ")
         bonus = 1
-        if (content[1] === "all"){
-            amount = user.dabs
-            choice = content[2]
-            bonus = 2
-        } else if (content[2] === "all") {
-            amount = user.dabs
+        if (content.length !== 3){
+            message.channel.send("Needs 2 arguments. Check $help betflip for" +
+                " more details.")
+            return 1
```

Comment:
No need for strange return values, just return. I ususally return the message.channel.send("Wrong") to save a line.